docs/learning/training/ontrack-comprehensive-setup.md

```diff
@@ -0,0 +1,91 @@
+# Ontrack Comprehensive Setup Guide
+
+- Windows
+
+## Windows (using WSL2)
+
+### Step 1: Set up WSL2
+
+The first step in setting up your development environment will be to set up Windows Subsystem for Linux (WSL) and Ubuntu. WSL allows Linux distributions to run within the Windows OS. Visit this [website](https://docs.microsoft.com/en-us/windows/wsl/install) for more information.
+
+To quickly set up WSL2 with Ubuntu, follow the tutorial on [ubuntu.com](https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-10#1-overview), OR (to paraphrase):
+
+1. Run Windows PowerShell as administrator (search "Powershell" in the windows search bar and right-click 'run as administrator') and use the following command:
+
+   ```powershell
+   wsl --install -d ubuntu
+   ```
+
+   This will set up WSL2 and the Ubuntu distribution.
+
+2. Restart your computer.
+
+3. Open Ubuntu (search for the app on your computer) and follow the first time setup instructions to create a username and password.
+
+   **NOTE: If you experience errors during this step, it may be because virtualization is disabled in your BIOS. To fix this will be different depending on your computer/CPU.**
+
+4. To make sure you have the latest updates for Ubuntu, use the following commands:
+   ```sh
+   sudo apt update
+   ```
+   then
+   ```sh
+   sudo apt upgrade
+   ```
+   You now have a Linux distribution running within Windows!
+
+### Step 2: Download and install Docker Desktop
+
+Docker Desktop is a software that is used to simplify the development of Doubtfire/OnTrack. Docker removes the need to install native tools used within the project.
+
+1. Download Docker Desktop from the [Docker website](https://docs.docker.com/desktop/windows/install/).
+
+### Step 3: Set up the OnTrack repository (following [CONTRIBUTING.md]()) (this step is WIP)
+
+Finally we can clone a local version of Doubtfire/OnTrack.
+
+On github, fork [doubtfire-deploy](), [doubtfire-api](), and [doubtfire-web](https://github.com/thoth-tech/doubtfire-web).
+**Make sure to fork the THOTH TECH branch.**
+
+#### note: Should we still be following the fork workflow?
+
+Open your Ubuntu terminal and create a directory for Doubtfire.
+
+```sh
+mkdir Doubtfire
+```
+
+cd into this directory
+
+```sh
+cd Doubtfire
+```
+
+clone your doubtfire-deploy
+
+```sh
+git clone --recurse-submodules https://github.com/YOUR_USERNAME/doubtfire-deploy
+```
+
+WIP!!!!!
```

Comment:
**question:** @lachfoy is this PR still a draft? Are you still working on these content?

Reply:
Yes - It is a work in progress. Should I instead finalize before creating a PR?

Reply:
Thanks for clarifying @lachfoy. 

> Should I instead finalize before creating a PR?

Not really. According to our code review [guideline](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/git-contribution-guide.md#code-review-guidelines), we would prefer to have a **draft PR** created as soon as you can. I would suggest setting this as a Draft so people know that you are still working on it.

Reply:
Ah, sorry! I'm still pretty new to GitHub still, and I actually had no idea that was a thing. I will do that.